src/main/java/br/com/magic/application/entity/mapper/JuniorCardMapper.java

```diff
@@ -9,4 +9,5 @@
 public interface JuniorCardMapper {
     List<JuniorCardDTO> toDto(List<JuniorCard> cards);
     List<JuniorCard> toEntity(List<JuniorCardDTO> cards);
+    JuniorCardDTO dto (JuniorCard card);
```

Comment:
```suggestion
    JuniorCardDTO toDto (JuniorCard card);
```
poderia o nome para "toDto", para seguir o padrão